src/Time.php

```diff
@@ -483,8 +483,8 @@ private function syncServiceOnlineDataToLocalCache($service)
         if (empty($data) === false) {
             $this->saveServiceDataCache($service, $data);
             $message .= '- ' . ucfirst($service) .': data cached'."\r\n";
-            } else {
-                $message .= '- ' . ucfirst($service) .': cannot cache data'."\r\n";
+        } else {
```

Comment:
__[phpmd]__
* The method syncServiceOnlineDataToLocalCache uses an else expression. Else is never necessary and you can simplify the code to work without else.

<!-- Comment by [SideCI](https://sideci.com) -->